core/commands.go

```diff
@@ -53,6 +53,17 @@ var (
 		Arity:    2,
 		KeySpecs: KeySpecs{BeginIndex: 1},
 	}
+	getDelCmdMeta = DiceCmdMeta{
+		Name: "GETDEL",
+		Info: `GETDEL returns the value for the queried key in args
+		The key should be the only param in args
+        If the key exists, it will be deleted before its value is returned.
```

Comment:
nit: Can you fix this indentation?

Reply:
earlier i try to fix it, it was coming correct in my compiler, let me check my compiler setting for indentation.